docs/pages/reference/helm-reference/includes/zz_generated.teleport-kube-agent.mdx

```diff
@@ -0,0 +1,1627 @@
+
+## `roles`
+
+| Type | Default |
+|------|---------|
+| `string` | `"kube"` |
+
+`roles` is a comma-separated list of services which should be enabled
```

Comment:
"should be enabled" sounds like it's nondeterministic—I'd recommend "will be enabled".